ci/Dockerfile

```diff
@@ -36,3 +36,4 @@ RUN cargo install cargo-llvm-cov cargo-nextest cargo-udeps cargo-hakari cargo-so
 
 ENV RUSTC_WRAPPER=sccache
 ENV SCCACHE_BUCKET=ci-sccache-bucket
+ENV CARGO_INCREMENTAL=0
```

Comment:
Why do we need this extra env? 🤔

Reply:
Well as @xxchan mentioned we are building the docker image from scratch

Reply:
Although, I'm not sure if we use this dockerfile to build release binaries. If we do it is even better since the docker build is always from scratch.

Reply:
> Why do we need this extra env? 🤔

So that we don’t need to set every ci-* profile…🤪 I just didn’t delete the profile settings

Reply:
> Although, I'm not sure if we use this dockerfile to build release binaries/performance pipeline. If we do it is even better since the docker build is always from scratch.

Didn’t thought about that, but yes it can be enabled. It’s a free optimization 🤔

Reply:
Yes, I'm saying it's in fact this env var which would set the incremental build to true, as the benchmarking build relies on the docker image built from this file.